src/lowlevel/protobuf/messages.js

```diff
@@ -21,7 +21,16 @@ export class Messages {
     const messagesByType: MessageArray<number> = {};
     Object.keys(messages.MessageType).forEach(longName => {
       const typeId = messages.MessageType[longName];
-      const shortName = longName.split(`_`)[1];
+
+      let shortName = longName.split(`_`)[1];
+
+      // hack hack hack :(
+      const indexOfDeprecated = longName.indexOf(`Deprecated`);
+      if (indexOfDeprecated === -1) {
+        shortName = longName.split(`_`)[1];
```

Comment:
Just a nitpick, if `indexOfDeprecated === -1` this assignment is not necessary because the correct value has already been assigned.

Maybe this is enough?

```
if (indexOfDeprecated >= 0) {
    shortName = longName.substr(indexOfDeprecated);
}
```